poi-ooxml/src/main/java/org/apache/poi/xssf/streaming/SXSSFSheet.java

```diff
@@ -63,7 +63,7 @@ protected SXSSFSheet(SXSSFWorkbook workbook, XSSFSheet xSheet, int randomAccessW
         setRandomAccessWindowSize(randomAccessWindowSize);
         try {
             _autoSizeColumnTracker = new AutoSizeColumnTracker(this);
-        } catch (InternalError e) {
+        } catch (Throwable e) {
```

Comment:
this code should not affect poi-scratchpad - but I agree that Throwable is a safer type to catch here